GitUI/UserControls/MainThreadScheduler.cs

```diff
@@ -19,7 +19,15 @@ public override IDisposable Schedule<TState>(TState state, TimeSpan dueTime, Fun
                 async () =>
                 {
                     await Task.Delay(normalizedTime, token);
```

Comment:
The context does not need to be captured here.
```csharp
await Task.Delay(normalizedTime, token).ConfigureAwait(false);
```

Reply:
I used `ConfigureAwaitRunInline`, a new extension from vs-threading 15.8. For now I had to define a copy of the extension in GitExtUtils, but I'm going to start the process of updating Git Extensions to use the new version of vs-threading.